### PR TITLE
[HotFix]Add symfony/form conflict

### PR DESCRIPTION
**Pull request by Tomanhez**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

### CODE REVIEW

**Comment by pamil** [comment]
Is this bug reported to Symfony?

**Comment by Tomanhez** [comment]
Nope. I didn't do that yet, I try to investigate what exactly is broken.

File: composer.json [hunk 1]
```diff
@@ -188,7 +188,7 @@
         "api-platform/core": "^2.5.5",
         "doctrine/inflector": "^1.4",
         "sylius/grid-bundle": "1.7.4",
-        "symfony/form": "4.4.9",
+        "symfony/form": "^4.4.9",
```
Comment (GSadee) [inline composer.json:191]: Maybe we should add:
```suggestion
        "symfony/form": "4.4.9|4.4.10",
```

**Comment by GSadee** [comment]
And I guess that it should be opened on 1.7 branch

**Comment by GSadee** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "fix-broken-build"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

**Comment by lchrusciel** [comment]
Thank you, Tomasz! :1st_place_medal:

**Pull request merged**